Userland/Libraries/LibWeb/CSS/Enums.json

```diff
@@ -433,5 +433,9 @@
     "pre",
     "pre-line",
     "pre-wrap"
+  ],
+  "direction": [
+    "ltr",
+    "rtl"
   ]
```

Comment:
Please move this up so that the file is sorted alphabetically.

Also, please modify Properties.json to make the `direction` property use this directly:

```diff
  "direction": {
    "animation-type": "none",
    "inherited": true,
    "initial": "ltr",
-    "valid-identifiers": [
-      "ltr",
-      "rtl"
-    ]
+    "valid-types": [
+      "direction"
+    ]
  },
```